modules/web/src/assets/config/changelog.json

```diff
@@ -1,4 +1,185 @@
 {
   "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.22.md",
-  "entries": []
+  "entries": [
+    {
+        "category": "action-required",
+        "description": "KubeVirt cloud provider support is GA now. Manual migration is required, check out the changelog for details."
+    },
+    {
+        "category": "action-required",
+        "description": "OperatingSystemProfiles and OperatingSystemConfigs have been moved to user clusters. CustomOperatingSystemProfiles have been introduced to maintain custom profiles at the Seed level."
+    },
+    {
+        "category": "action-required",
+        "description": "Helm support for Applications has been updated to 3.11. Due to possible CVEs, the template function getHostByName is disabled by default and needs to be enabled explicitly."
+    },
+    {
+        "category": "added",
+        "description": "Support for Kubernetes 1.25 and Kubernetes 1.26."
+    },
+    {
+        "category": "added",
+        "description": "Add a web terminal to directly interact with user cluster APIs."
+    },
+    {
+        "category": "added",
+        "description": "Support for importing KubeOne clusters."
+    },
+    {
+        "category": "added",
+        "description": "Add group information to OIDC kubeconfig."
+    },
+    {
+        "category": "added",
+        "description": "Preset availability can be limited to specific projects."
+    },
+    {
+        "category": "added",
+        "description": "Support for external CCM & CSI drivers in AWS. New clusters default to this, existing clusters need to be CCM migrated."
+    },
+    {
+        "category": "added",
+        "description": "Support for GCP CSI Driver in Kubernetes 1.25+ clusters."
+    },
+    {
+        "category": "added",
+        "description": "Support for using server groups with OpenStack."
+    },
+    {
+        "category": "added",
+        "description": "Support for DigitalOcean external CCM."
+    },
+    {
+        "category": "added",
+        "description": "Add Canal CNI v3.24."
+    },
+    {
+        "category": "added",
+        "description": "Add Cilium CNI v1.13.0."
+    },
+
```

Comment:
Are you sure those empty lines don't affect JSON parsing?

Reply:
I tried it by running the dashboard locally and it was fine, I was able to see the full changelog via it.